kubectl-metalk8s/pkg/salt/client.go

```diff
@@ -160,7 +160,7 @@ func (self *Client) PollJob(
 	ctx context.Context, jobId string,
 ) (map[string]interface{}, error) {
 	jobLogger := self.logger.WithValues("Salt.JobId", jobId)
-	jobLogger.Info("polling Salt job")
+	jobLogger.V(3).Info("polling Salt job")
```

Comment:
Ditto, use a constant here.
If 3 is used only for Salt log, then could be called `SALT_LOG_LEVEL`